### DISCUSSION TITLE
Unsubscribe Ability?

### DISCUSSION THREAD

**Discussion by toofyspooder** [XilinJia/Podcini#100] [Q&A]
Wasn't able to find an option in the app, or a discussion on here related to this.

Is there the ability to unsubscribe from a podcast in the app? I see a way to removebthe podcast, but it honestly always just pops back into the app. Haven't been able to find a way to remove it completely.

**Comment by XilinJia** [comment]
remove means unsubscribe. There are three places to do it: from the menu of FeedInfo and FeedEpisodes, and from the multi-selection menu of Subscriptions.  Once removed, it's gone.

Can you explain about "it honestly always just pops back into the app"?

**Reply by toofyspooder** [reply]
Sorry for not replying on this.

What was happening is I would remove a podcast and it would disappear from Subscriptions as expected. The next time I opened the app it would return. Remove it again, and again it would return. Maybe due to syncing with gpoddersync on my nextcloud?

Unfortunately ever since a couple updates ago I have love all my subscriptions completely. All listening history, everything. Which is a little frustrating.

**Comment by XilinJia** [comment]
Very sorry to hear your experience.

Unsubscribe a podcast and then see it comes back is not something Podcini does. 

Syncing with the servers was inherited from AntennaPod but has not been well maintained in Podcini.  My prior experiences in other projects on syncing with things like nextcloud have never been good, and I don't have such a setup to test with Podcini.  I've long debated what to do about it.  Its functioning so far has relied on the previous code and some contributions from others.  I hope it's not caused the disastrous problem like yours to many people.

#49

**Reply by toofyspooder** [reply]
Anything I can do to help? Would love to have this ability it Podcini as I do love the app. Have very limited coding experience but even just testing for you I am willing and able to do.
